packages/server/src/routes/cron/notifyOnUnminted.ts

```diff
@@ -89,11 +90,16 @@ export const notifyOnUnminted = (
             notificationHistoryCollection.doc(member.id)
           );
           if (await shouldNotify(auth, member, memberNotificationHistory)) {
-            const title = `Mint ${calculateMaxMintableForMember(member)
-              .round(2, 0)
-              .toString()} Raha before it's too late!`;
-            const body =
-              "Your mintable amount will be capped at 40 Raha on November 15th. Tap this notification to mint now and avoid losing Raha!";
+            const isMintCapped = isPastMintCapTransitionDate();
+            const maxMintable = calculateMaxMintableForMember(
+              member
+            ).toString();
+            const title = isMintCapped
+              ? `Mint ${maxMintable} Raha now!`
+              : `Mint ${maxMintable} Raha before it's too late!`;
+            const body = isMintCapped
+              ? "You can't get any more basic income until you've minted. Tap to mint now and keep receiving your Raha!"
+              : "Your mintable amount will be capped at 40 Raha on November 15th. Tap this notification to mint now and avoid losing Raha!";
```

Comment:
nit: 11/15 or Nov 15?

Reply:
addressed in //notify-on-unminted branch